src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public String goWork() {
+        return "Bulldozer started its work!";

```

Comment:
The message returned here should be more informative and indicate that it is a Bulldozer that is working. Consider including the type of machine in the message.